src/crypto/tls/common.go

```diff
@@ -442,6 +442,9 @@ type ClientSessionCache interface {
 // RFC 8446, Section 4.2.3.
 type SignatureScheme uint16
 
+const (
+	undefinedSignatureScheme SignatureScheme = 0x0000
+)
```

Comment:
nit: This is a potential rebase conflict risk. Consider moving to `delegated_credentials.go`.